internal/node_install.bzl

```diff
@@ -86,14 +90,18 @@ package(default_visibility = ["//visibility:public"])
 exports_files(['yarn.sh'])
 alias(name = "yarn", actual = ":yarn.sh")
 """)
+  if ctx.os.name.lower().find("windows") != -1:
+    node = Label("@nodejs//:node.exe")
```

Comment:
why are you not using the alias defined above, but special case node.exe for windows?

Reply:
the alias isn't resolved in skylark, if you `ctx.path(Label(alias))` you get the location of the alias not the target.
```
Not a file: /usr/local/google/home/alexeagle/.cache/bazel/_bazel_alexeagle/c410d6070ef2d359d26ac4aeb886e1ea/external/nodejs/node
```
I guess the alias resolution happens in the loading phase?

Reply:
Maybe something to task? Also, why do you have the alias then in the first place?